database/migrations/2023_10_30_115558_update_nft_thumbnail_to_scaled.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+return new class() extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        DB::table('nfts')
+            ->whereRaw("extra_attributes->'images'->>'thumb' LIKE '%thumbnailv2%'")
+            ->orWhereRaw("extra_attributes->'images'->>'small' LIKE '%thumbnailv2%'")
+            ->orWhereRaw("extra_attributes->'images'->>'large' LIKE '%thumbnailv2%'")->get()->each(function ($nft) {
+                $extraAttributes = json_decode($nft->extra_attributes);
```

Comment:
for this I meant an actual query that updates the values and doesn't rely on eloquent models. You rewrote it now, but it will contain the same amount of models as before as they basically all use that thumbnail